src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
+  font-family: Roboto, serif;
+  font-weight: 500;
+  font-style: normal;
+  background-color: darkblue;
+}
+
+header {
+  background-color: white;
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  display: flex;
+}
+
+img {
+  display: block;
+}
+
+nav {
+  display: flex;
+}
+
+.nav-list {
+  display: flex;
+  list-style-type: none;
+  text-transform: uppercase;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-link {
+  display: flex;
+  font-size: 12px;
+  line-height: 60px;
+  text-decoration: none;
+  color: #000;
+  margin-left: 20px;
+  white-space: nowrap;
+  position: relative;
+  align-items: center;
+}
+
+.nav-list li:first-child .nav-link {
+  margin-left: 0;
+}
+
+:hover {
```

Comment:
The `:hover` selector is too generic and might apply to unintended elements. Consider specifying the element it should apply to, such as `.nav-link:hover`, to avoid unexpected styling changes.